schema.development.kf

```diff
@@ -486,21 +486,21 @@ action share_credential (
     add_credential(
         $id
       , $issuer_auth_public_key
-      , $encryption_public_key
+      , $encryptor_public_key
       , $content
       , $public_notes
       , $public_notes_signature
       , $broader_signature
     );
-    INSERT INTO shared_credentials (original_id, duplicate_id) VALUES ($original_credential_id, $id);
+    INSERT INTO shared_credentials (original_id, copy_id) VALUES ($original_credential_id, $id);
 }
 
-// Who is using this? Could very well be nobody.
+// It can be used with EVM-compatible signatures only
 action share_credential_through_dag (
```

Comment:
Did we miss `dag_owner` and `dag_grantee`, or did you have a reason to keep them with the old style?